src/zen/forks/fork4_nulltransactionfork.cpp

```diff
@@ -21,7 +21,7 @@ NullTransactionFork::NullTransactionFork()
                                      "zsxWnyDbU8pk2Vp98Uvkx5Nh33RFzqnCpWN"
                                      }},
                                 {CBaseChainParams::Network::REGTEST,{
-                                     "zrKmSdqZKZjnARd5e8FfRg4v1m74X7twxGa"
+                                     "zr9kPMxW6zVgKQnAXZtbMJDv2FYoTrvyhsN"
```

Comment:
These are p2sh, for generatong them use is made of some cli cmd:
 src/zen-cli -regtest getnewaddress ---> <t_addr>
 src/zen-cli -regtest validateaddress <t_addr> ---> <scriptPubKey>
 src/zen-cli -regtest decodescript <scriptPubKey> ---> <p2sh>

Reply:
I think it would make sense to also commit the private keys used to generate these addresses (in a comment). We could then spend from one of these addresses as part of a RPC test if the need ever arises in the future.

Reply:
Ok, done